zone/gm_commands/npcspecialattk.cpp

```diff
@@ -2,15 +2,97 @@
 
 void command_npcspecialattk(Client *c, const Seperator *sep)
 {
-	if (c->GetTarget() == 0 || c->GetTarget()->IsClient() || strlen(sep->arg[1]) <= 0 || strlen(sep->arg[2]) <= 0) {
+	if (!c->GetTarget() || !c->GetTarget()->IsNPC()) {
+		c->Message(Chat::White, "You must target an NPC to use this command.");
+		return;
+	}
+
+	int arguments = sep->argnum;
+	if (!arguments) {
 		c->Message(
 			Chat::White,
-			"Usage: #npcspecialattk *flagchar* *permtag* (Flags are E(nrage) F(lurry) R(ampage) S(ummon), permtag is 1 = True, 0 = False)."
+			"Usage: #npcspecialattk [Flag] [Permanent] - Set an NPC's special attack flag, use permanent to set permanently."
 		);
+		c->Message(
+			Chat::White,
+			"Usage: #npcspecialattk help - List all the special attack identifiers"
+		);
+		return;
+	} else if (arguments == 1) {
+		if (!strcasecmp(sep->arg[1], "help")) {		
+			c->SendPopupToClient(
+				"Special Attack Flags",
```

Comment:
Using flags like this is deprecated, not sure we want to even encourage the use at this point

Reply:
Can remove the command altogether if you'd prefer and just add to a Wiki page for legacy purposes.